docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -858,8 +895,11 @@ Each `coin` can be enabled only once, and in either Electrum or Native mode. The
 For utxo-based coins the daemon of this blockchain must also be running on the user's machine for `enable` to function. 
 
 ::: tip Note
+
 The MM2 node's coin address needs to be imported manually into the coin daemon using the [importaddress](../../../basic-docs/smart-chains/smart-chain-api/wallet.html#importaddress) call.
 
+Native mode for QRC20-based coins is in development yet.
```

Comment:
`QRC20-based coins` - same here, better to use `QRC20 tokens`.

Reply:
Done